cmd/protoc-gen-go-grpc/go.mod

```diff
@@ -1,5 +1,5 @@
 module google.golang.org/grpc/cmd/protoc-gen-go-grpc
 
-go 1.19
+go 1.21
```

Comment:
Did this happen because of a dependency?  Is there some way to find out which one?  Maybe a `go` command?